src/sagas/wallet.js

```diff
@@ -249,7 +250,7 @@ export function* loadTokens() {
     }, []);
 
   // We don't need to wait for the metadatas response, so just fork it
```

Comment:
Should this comment be updated?

Reply:
Actually, the comment made more sense now than it did before the change, `spawn` is just a "detached" version of `fork`

But I updated it to make the difference clearer on 19fe352ea625c1ef35c58e62df1865497d7629ee